src/qt/test/wallettests.cpp

```diff
@@ -156,7 +156,7 @@ void TestGUI(interfaces::Node& node)
 
         WalletRescanReserver reserver(wallet.get());
         reserver.reserve();
-        CWallet::ScanResult result = wallet->ScanForWalletTransactions(locked_chain->getBlockHash(0), {} /* stop_block */, reserver, true /* fUpdate */);
+        CWallet::ScanResult result = wallet->ScanForWalletTransactions(Params().GetConsensus().hashGenesisBlock, 0 /* block height */, {} /* max height */, reserver, true /* fUpdate */);
```

Comment:
Why `auto locked_chain = wallet->chain().lock();` is still needed?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/17954#discussion_r395737849

> Why `auto locked_chain = wallet->chain().lock();` is still needed?

Good catch, simplified this code